### PR TITLE
sidebar more information styling

### LINKED ISSUE #93: Sidebar More Information Styling
**Is your feature request related to a problem? Please describe.**
The sidebar will render more information about the selected pokemon

**Describe the solution you'd like**
Modify the sidebar to have a set minimum height and enable scroll functionality.

### PR DESCRIPTION
**Pull request by gonzalespaulb**
## Changes
1. More information section in the sidebar styled
2. Added a grid to house the game indices

## Purpose
Reorganize the information passed to the sidebar

Closes #93

### CODE REVIEW

**Review by connielion — APPROVED** [review]
Looks good! And nice comments!

File: pokeapp/src/components/Sidebar.js [hunk 1]
```diff
@@ -116,26 +118,74 @@ const SideBar = ({
...
+          <div className="height-weight">
+                <div className="height">
+                  <h4>Height:</h4>
+                  <h5>{selectedPokemon.height}</h5>
+                </div>
+                <div className="weight">
+                  <h4>Weight:</h4>
+                  <h5>{selectedPokemon.weight}</h5>
+                </div>
+          </div>
```
Comment (Gjhnsn) [inline pokeapp/src/components/Sidebar.js:161]: This whole section with the pokemon info is really well organized

File: pokeapp/src/components/Sidebar.js [hunk 2]
```diff
@@ -33,6 +33,9 @@ const SideBar = ({
   const [showPokemon, setShowPokemon] = useState(false);
   const [pokeNameList, setPokeNameList] = useState([]);
 
+  const [videoGame, setVideoGame] = useState(`No Game Selected`);
```
Comment (BlackCubes) [inline pokeapp/src/components/Sidebar.js:36]: Videogame 👀

File: pokeapp/src/components/Sidebar.js [hunk 3]
```diff
@@ -116,26 +118,74 @@ const SideBar = ({
...
+                {isMoreInfo ? gameMapper(selectedPokemon.games, setVideoGame) : null}
+                {!selectedPokemon.games.length && isMoreInfo ? (
+                  <p>This Pokemon has not been in any games</p>
+                ) : null}
+            </div>
+          </div>
+
+{/* ---------------------------------------------------------------------------- BUY, SELL, AND MORE INFO BUTTON */}
+
+          <div className={!isMoreInfo ? "btn-container" : "btn-container-more-info"}>
```
Comment (JorgeLVilla) [inline pokeapp/src/components/Sidebar.js:188]: Love the organization!

**Review by Gjhnsn — APPROVED** [review]
Looks great!

**Review by JorgeLVilla — APPROVED** [review]
Nice job Paul!

**Review by BlackCubes — APPROVED** [review]
Everything looks really good! Can't wait!

**Review by JayDampitan — APPROVED** [review]
Nice touch with the game indices and how it shows the name when you hover over it.

**Pull request merged**